Promact.Oauth.Server/src/Promact.Oauth.Server/Services/MessageServices.cs

```diff
@@ -39,9 +40,9 @@ public void SendEmail(string email, string subject, string message)
             using (var smtp = new SmtpClient())
             {
                 _logger.LogInformation("Smtp Connect");
-                smtp.Connect(_appSettings.Value.Host, _appSettings.Value.Port, _appSettings.Value.SslOnConnect == true ? MailKit.Security.SecureSocketOptions.SslOnConnect : MailKit.Security.SecureSocketOptions.None);
+                smtp.Connect(_appSettings.Host, Convert.ToInt32(_appSettings.Port), _appSettings.SslOnConnect == true ? MailKit.Security.SecureSocketOptions.SslOnConnect : MailKit.Security.SecureSocketOptions.None);
```

Comment:
@chintans right now first convert to int then set to port like below code 
` int port = Convert.ToInt32(_appSettings.Port);
  smtp.Connect(_appSettings.Host, port, _appSettings.SslOnConnect == true ? MailKit.Security.SecureSocketOptions.SslOnConnect : MailKit.Security.SecureSocketOptions.None);`